debs/SPECS/3.12.0/wazuh-api/debian/control

```diff
@@ -0,0 +1,12 @@
+Source: wazuh-api
+Section: admin
+Priority: extra
+Maintainer: Wazuh, Inc <info@wazuh.com>
+Build-Depends: debhelper (>= 7.0.50~), nodejs (>= 4.6), npm, curl, gawk
+Standards-Version: 3.8.4
+Homepage: http://www.wazuh.com
+
+Package: wazuh-api
+Architecture: any
+Depends: ${shlibs:Depends}, npm, nodejs (>= 4.6.1), wazuh-manager (>= 3.11.0), wazuh-manager (<< 3.11.1), curl
```

Comment:
Please change `3.11` to `3.12` on `wazuh-api` control file.

Reply:
Done in 7f378b0